src/UserSpaceInstrumentation/include/UserSpaceInstrumentation/InjectLibraryInTracee.h

```diff
@@ -5,6 +5,7 @@
 #ifndef USER_SPACE_INSTRUMENTATION_INJECT_LIBRARY_IN_TRACEE_H_
 #define USER_SPACE_INSTRUMENTATION_INJECT_LIBRARY_IN_TRACEE_H_
 
+#include <dlfcn.h>
```

Comment:
I get it, but a automatic check would complain since this violates iwyu. Suggestion: make `RTLD_NOW` the default for `flag` parameter below.

Reply:
Good idea, thanks, I did just that.

Reply:
Actually I'll submit the change in a separate PR and remove RTLD_NOW from other call sites.

Reply:
I don't get it, what's the reason for adding the default `RTLD_NOW` value? This to seems to be a case where using a default argument reduces readability at the call sites.

Reply:
The main thing I want to do is include <dlfcn.h> here. Maybe "// IWYU pragma: keep" is what we need?

Reply:
I did just that, although I didn't mind the default parameter.